interfaces/inc/c_pal/gballoc_hl.h

```diff
@@ -11,6 +11,8 @@
 #include <stdint.h>
 #endif
 
+#include "malloc_multi_flex.h"

```

Comment:
>malloc_multi_flex [](http://example.com/codeflow?start=10&length=17)

I think this is definitely a special-case and we can just have callers include malloc_multi_flex as needed rather than having it in this header.